apps/desktop/desktop/src/main/java/bisq/desktop/common/utils/FileChooserUtil.java

```diff
@@ -31,6 +31,10 @@
 
 @Slf4j
 public class FileChooserUtil {
+
+    private static final boolean DIRECTORY_CHOOSER = true;
```

Comment:
Maybe better rename to IS_DIRECTORY_CHOOSER. When i read the usage I expected that its an file chooser object.